src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java

```diff
@@ -169,7 +169,7 @@ public void testFailedWorkingBalancesAdjustmentComputation() throws InterruptedE
         InputStream targetNetPositionsIStream = new FileInputStream(ResourceUtils.getFile("classpath:failingTargetNetPositions.json"));
         BalanceComputationResult balanceComputationResult = balancesAdjustmentService.computeBalancesAdjustment(testNetworkId, balanceComputationParameters, targetNetPositionsIStream, true, false);
         assertEquals(BalanceComputationResult.Status.FAILED, balanceComputationResult.getStatus());
-        assertEquals(11, balanceComputationResult.getIterationCount());
+        assertEquals(10, balanceComputationResult.getIterationCount());
```

Comment:
why balanceComputationResult.getIterationCount() returns 10 and not 11 as before ?

Reply:
before, balances adjustment was using powsybl core 3.3.0 and open load flow 0.3.0
We are now in 3.5.0 and 0.5.0, and the load flow results are not exactly the same ...